documentation/docs/checkout-push.mdx

```diff
@@ -23,14 +23,14 @@ activate Acquirer
 Acquirer->>Acquirer: createInvoice(invoice)
 Acquirer-->>Receiver: 201 paid=false
 deactivate Acquirer
-Receiver-xSender: [ send invoice_url out of band (PaymentRequest, QR Code) ]
+Receiver-xSender: [ send invoice_url out of band (PaymentRequest, QR Code, Redirect) ]
```

Comment:
Is it a goal for Open Payments to also formalize how the invoice URL would be exchanged using these mechanisms? Specifically, using `PaymentHandler` or a QR code?

Reply:
Possibly. Not in this update though 😄